src/services/helper/entity.js

```diff
@@ -51,40 +55,30 @@ module.exports = class EntityHelper {
 	 * @returns {JSON} - Entity updted response.
 	 */
 
-	static async update(bodyData, _id, loggedInUserId) {
-		bodyData.updatedBy = loggedInUserId
-		bodyData.updatedAt = new Date().getTime()
+	static async update(bodyData, id, loggedInUserId) {
+		bodyData.updated_by = 1 || loggedInUserId
```

Comment:
[DISCUSS]

Reply:
The temporary solution has been removed.